### DISCUSSION TITLE
Temperature, Water, Doors etc. sensors not visible or listed in homekit app

### DISCUSSION THREAD

**Discussion by DocIOTUpdate** [homebridge-simplisafe3/homebridge-simplisafe3#406] [Q&A] [answered]
Previously I once you log in to SimpliSafe via the Homebridge Plugin you could see in HomeKit:

- Previously Temperature sensors being visible and their reported temperature status > Today no sensors are listed
- Previously you could see door open/closed > Today only Open doors appear as needed
- Previously Water Sensors showed up as normal > Today the Water icon is shown but lists no sensors unless there is a problem

All I see now is the tile for the SimpliSafe System Alarm which always shows whether if it is either on or off.

How come the Plugin functionality has reduced so dramatically. Can it no longer pick up and list devices persistently and their status?

Please can you help as this is driving me nuts! 

Many thanks in advance.

**Comment by shamoon** [comment]
I hadn't noticed this but they seem to be HomeKit / iOS changes, nothing to do with this plugin.

Seems mostly like the new Home app changes only display sensors that are somehow abnormal, open, triggered etc. If you go into the room(s) I think you'll find the sensors are still there.

<img width="337" alt="Screenshot 2023-10-09 at 1 28 31 PM" src="https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/assets/4887959/aa8ed276-0324-45f9-8259-a4bb8048d37c">
<img width="165" alt="Screenshot 2023-10-09 at 1 28 26 PM" src="https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/assets/4887959/cb2c1af1-07fb-4cb6-a69b-197c414af467">
<img width="481" alt="Screenshot 2023-10-09 at 1 28 21 PM" src="https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/assets/4887959/0296ea41-2aa3-4250-b93a-5fe511d7b24b">

**Reply by DocIOTUpdate** [reply]
Nope the sensors in the past would show up but now they do not. They do for Dyson, Airthings etc. but not SimpliSafe anymore. Is this due to the matter upgrade or something?

**Reply by shamoon** [reply]
I dont know what you mean by matter update, if you mean HomeKit's support of matter, I dont think so since thats been available for a while. SS do not support matter.

As you can see from the screenshots, the sensors are still available to HomeKit.

Furthermore, if you enable debug and restart the logs will show you extensive information about what sensors are found etc.
`Discovered sensor 'Water Heater' from SimpliSafe: {"flags":{"offline":false,"lowBattery":false,"swingerShutdown":false},"serial":"","type":9,"name":"Water Heater","setting":{"alarm":1},"status":{"triggered":false},"deviceGroupID":0}`

Homebridge UI X also shows accessories itself:
<img width="173" alt="Screenshot 2023-10-09 at 1 43 58 PM" src="https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/assets/4887959/d475743f-de40-42e7-a878-d9b8a9558988">

Again, things havent changed on the plugin side.

**Reply by DocIOTUpdate** [reply]
See no SimpliSafe sensors:
![2023-10-09 21 48 25](https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/assets/107815892/46c05357-4469-4530-8fb3-544d4e24b9be)
![2023-10-09 21 49 01](https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/assets/107815892/1a7bd6ff-4c9d-413c-a88a-066790679ae5)

**Reply by shamoon** [reply]
I dont have SS temp sensors but the leak sensors do not show under water.

If you re-read my comment above, I have to go into the room where they are and theres a Leak Sensor category
<img width="108" alt="Screenshot 2023-10-09 at 1 51 36 PM" src="https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/assets/4887959/6beb8be6-d621-4eff-ad77-4e9eb5ac6201">

**Reply by shamoon** [reply]
Entry sensors too
<img width="235" alt="Screenshot 2023-10-09 at 1 52 39 PM" src="https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/assets/4887959/e7412c41-3507-4556-886d-ffe1aef34294">

**Reply by DocIOTUpdate** [reply]
This is the cloak room which has a temperature sensor that shows 22 degrees in SimpliSafe. This is what i see in HomeKit now:
![2023-10-09 21 54 46](https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/assets/107815892/493450fe-07c5-4c41-804d-81e2fb6c53ce)

**Reply by shamoon** [reply]
Please see the comment above to investigate: https://github.com/homebridge-simplisafe3/homebridge-simplisafe3/discussions/406#discussioncomment-7234287

**Comment by DocIOTUpdate** [comment]
10/9/2023, 9:18:26 PM] [Home Alarm] 3 errors were received from the SimpliSafe API while refreshing sensors in the last 5 minutes. These can usually be ignored if everything is working. Otherwise, enable debug logging for the plugin and restart to see detailed output.
[10/9/2023, 9:43:41 PM] [Home Alarm] 2 errors were received from the SimpliSafe API while refreshing sensors in the last 5 minutes. These can usually be ignored if everything is working. Otherwise, enable debug logging for the plugin and restart to see detailed output.

[10/9/2023, 10:17:28 PM] [Home Alarm] SimpliSafe real time events connected.

I tried debug and waited and nothing showed up. Also renamed the Sensor instances in SimpliSafe thinking maybe that was in. However all that is show is the Alarms and that is it - no sensors unless they are tripped i.e. no Temperature sensors shown.

**Reply by shamoon** [reply]
You have to restart after enabling debug.

**Comment by DocIOTUpdate** [comment, marked as answer]
Found them - Clear the Cache of Homebridge and it resets the sensors! Sorted it!!
